content/applications/finance/accounting/payments/internal_transfers.rst

```diff
@@ -77,38 +72,30 @@ Automated booking - Bank journal (BANK B)
      - **$1,000**
 
 .. note::
-   There are one outstanding payment and one outstanding receipt pending in your two bank account
+   There is one outstanding payment and one outstanding receipt pending in your two bank account
    journals, because the bank statement confirming the sending and receiving of the money has not
    been booked yet.
 
-.. image:: internal_transfers/outstanding-payments-receipts.png
-   :align: center
-   :alt: Outstanding Payments/Receipts pending bank statement booking
+Internal transfer reconciliation
```

Comment:
Is this whole section necessary? Is there anything in the reconciliation process that's specific to internal transfers? 
If not, I would maybe just add a sentence at the end of the previous section explaining that reconciliation is done as usual + keep the seealso linking to the reconciliation doc page.